### PR TITLE
Improve mark function

### PR DESCRIPTION
**Pull request by awerebea**
Use the base name of the current directory as the default mark name.

### CODE REVIEW

File: fzf-marks.plugin.bash [hunk 1]
```diff
@@ -50,7 +50,11 @@ function _fzm_setup_completion {
 
 function mark {
     local mark_to_add
-    mark_to_add="$* : $(pwd)"
+    if [ -z "$1" ]; then
```
[the remark this answers is not in the document]
Comment (awerebea) [reply fzf-marks.plugin.bash]: Totally agree. Added [commit](https://github.com/urbainvaes/fzf-marks/pull/62/commits/e5bafada2379e137b7b6837aece19686687df4b2) with suggested improvements.
Thanks!

[the remark this answers is not in the document]
Comment (akinomyoga) [reply fzf-marks.plugin.bash]: Thanks!

**Comment by urbainvaes** [comment]
Many thanks! And sorry, I haven't had much time to keep track of the issues these last few months. @akinomyoga , it seems that this is ready to be merged?

**Comment by akinomyoga** [comment]
@urbainvaes Yes, I think this seems ready to be merged!

**Review by akinomyoga — APPROVED** [review]

**Comment by urbainvaes** [comment]
Thank you!

**Pull request merged**